src/store/traction/ont/runs/state.js

```diff
@@ -0,0 +1,7 @@
+const state = {
+  currentRun: {
+    flowcell_attributes: [],
```

Comment:
does this need a default `instrument_name: ''` or state for when a new run is built?

Reply:
Could set new run / empty run as default state and when users go to new run just restore state to default

Reply:
It does do that in the `newRun` function in the action. But maybe these `flowcell_attributes` could be removed here

Reply:
@BenTopping Yep, not needed, I'll make `currentRun` here just an empty object 👍

Reply:
Could remove the newRun function and just have it as default state, but either way is fine